documentation/docs/features/online_search.md

```diff
@@ -1,17 +1,21 @@
 # Online Search
 
-By default, Khoj will try to infer which information-sourcing tools are required to answer your question. Sometimes, you'll have a need for outside questions that the LLM's knowledge doesn't cover. In that case, it will use the `online` search feature.
+Khoj will research on the internet to ground it's responses, when real-time or other information outside it's existing knowledge is requested. It will always show any online references it used to respond to your requests.
 
-For example, these queries would trigger an online search:
+By default, Khoj will try infer which information sources, it needs to read to answer your question. This can include reading your documents or researching information online. You can also explicitly trigger an online search by adding the `/online` prefix to your chat query.
```

Comment:
```suggestion
By default, Khoj will try to infer which information sources it needs to read to answer your question. This can include reading your documents or researching information online. You can also explicitly trigger an online search by adding the `/online` prefix to your chat query.
```